tsc.md5

```diff
@@ -1 +1 @@
-039b1a77099af5b6dadf49d2b4957db0  tsc
+0835e1311abde2650c33682d9dc31ad9  tsc
```

Comment:
Dit hoef je dus niet zelf meer te doen. Zal automatisch gebeuren zodra hij in main branch komt